translations.yml

```diff
@@ -58,6 +58,16 @@ parts:
       title: "Label for the link color setting"
       screenshot: "https://zendesk.box.com/s/ufzzcaedpey0zp92nozg2yzhytsrhd8f"
       value: "Link color"
+  - translation:
+      key: "txt.help_center_copenhagen_theme.visited_link_color_description"
+      title: "Description for the visited link color setting"
+      screenshot: "https://zendesk.box.com/s/ufzzcaedpey0zp92nozg2yzhytsrhd8f"
+      value: "Text color for visited link elements"
+  - translation:
+      key: "txt.help_center_copenhagen_theme.visited_link_color_label"
+      title: "Label for the visited link color setting"
+      screenshot: "https://zendesk.box.com/s/ufzzcaedpey0zp92nozg2yzhytsrhd8f"
```

Comment:
The screenshot doesn't seem to match the value? And would it be possible to add it to this folder so the translators can have access? See here

Reply:
Updated the screenshot.